test/cluster/test_group0_schema_versioning.py

```diff
@@ -312,7 +312,7 @@ async def test_upgrade(manager: ManagerClient):
     await wait_for_cql_and_get_hosts(cql, servers, time.time() + 60)
 
     logger.info("Creating keyspace and table")
-    async with new_test_keyspace(manager, "with replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 3}") as ks_name:
+    async with new_test_keyspace(manager, "with replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 2}") as ks_name:
```

Comment:
nit: add `test_upgrade:` to patch title

Reply:
Would this be enough? It's more in line with our convention to stop at the file level:

`test: cluster: test_group0_schema_versioning: Use smaller RF to respect rf-rack-validness`